infra/nnfw/cmake/packages/TensorFlowLiteGpu/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Tensorflow Lite GPU delegate library 2.8.0
 #
 
-set(LIB_TENSORFLOW_GPU_DELEGATE "TensorFlowGpu")
+set(LIB_TENSORFLOW_GPU_DELEGATE tflite-gpu-delegate)
```

Comment:
Q. Do we have a rule for naming library? (IHMO, the lib's name has `-` delimiter instead of `_` and I think it is awkward.)

But, to be honost, I don't care.

Reply:
No, we don't have rule about this.